tools/CleanUp_tool_powershell_script/CleanUp_tool.ps1

```diff
@@ -0,0 +1,45 @@
+﻿#CleanUp tool 1.0
+#Copyright (C) 2020 Microsoft Corporation
+#Tool to clean PowerToys settings inside AppData folder and registry
+
+$SettingsPath = $Env:LOCALAPPDATA + '\Microsoft\PowerToys'
```

Comment:
I think it makes more sense to define the variable between comment and if statement.

P. S.: I would recommend to you to declare the vars correct as strings:
`[String]$varname=value`